<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogCampo</class>
 <widget class="QDialog" name="DialogCampo">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>355</width>
    <height>251</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QComboBox" name="CB_CampoTipo">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>90</y>
     <width>91</width>
     <height>26</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>CHAR</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>INTF</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>DEC</string>
    </property>
   </item>
  </widget>
  <widget class="QLineEdit" name="T_CampoNombre">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>40</y>
     <width>231</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="CampoNuevo">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>210</y>
     <width>113</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Aceptar</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>20</y>
     <width>121</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Nombre del Campo</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>70</y>
     <width>121</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Tipo de Campo</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="SB_CampoLongitud">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>123</y>
     <width>48</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>130</y>
     <width>121</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Longitud del Campo</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="RB_CampoLlave">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>170</y>
     <width>111</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Llave Primaria</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
